representation/normalize.go

```diff
@@ -1,13 +1,29 @@
-package representer
+package representation
 
 import (
 	"go/ast"
 	"go/types"
 
+	"github.com/pkg/errors"
```

Comment:
Are you sure you need this package? I rarely see people using it nowadays because of the built-in wrapping.

Reply:
It has stack traces... If you want I can throw it out after I'm done 😄

Reply:
would prefer to have the stack traces until then... otherwise I have to put them back in for debugging.